hack/patches/011-recordevents-mesh.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/vendor/knative.dev/eventing/test/lib/recordevents/resources.go b/vendor/knative.dev/eventing/test/lib/recordevents/resources.go
+index 3b04ed74b..ae85dc0ce 100644
+--- a/vendor/knative.dev/eventing/test/lib/recordevents/resources.go
++++ b/vendor/knative.dev/eventing/test/lib/recordevents/resources.go
+@@ -86,6 +86,10 @@ func recordEventsPod(imageName string, name string, serviceAccountName string, r
+ 		ObjectMeta: metav1.ObjectMeta{
+ 			Name:   name,
+ 			Labels: map[string]string{"e2etest": string(uuid.NewUUID())},
++			Annotations: map[string]string{
```

Comment:
do we want thsi _always_? Or is this applied / used w/ a certain condition ?

Reply:
It is always applied but it will be effective only in mesh tests (aka when servicemeshmemberroll has knative-eventing namespace)

Reply:
which layer above?